src/SCRIPTS/BF/PAGES/pos_osd.lua

```diff
@@ -0,0 +1,161 @@
+
+x = margin
+y = yMinLim - tableSpacing.header
+
+fields[#fields + 1] = {              x = x,                        y = inc.y(tableSpacing.header), min = 0, max = (#items - 1), vals = { 1 }, upd = function(self) self.updateItems(self) end }
+labels[#labels + 1] = { t = nameCli, x = x + tableSpacing.col * 1, y = y }
+
+labels[#labels + 1] = { t = "POS",   x = x,                        y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "OP1",   x = x + tableSpacing.col,     y = y }
+labels[#labels + 1] = { t = "OP2",   x = x + tableSpacing.col * 2, y = y }
+labels[#labels + 1] = { t = "OP3",   x = x + tableSpacing.col * 3, y = y }
+
+fields[#fields + 1] = {              x = x,                        y = inc.y(tableSpacing.row), min = 0, max = 2047, vals = { 2, 3 } }
+fields[#fields + 1] = {              x = x + tableSpacing.col,     y = y, min = 0, max = 1, vals = { 1, 2 }, table = { [0] = "OFF", "ON" } }
+fields[#fields + 1] = {              x = x + tableSpacing.col * 2, y = y, min = 0, max = 1, vals = { 1, 2 }, table = { [0] = "OFF", "ON" } }
+fields[#fields + 1] = {              x = x + tableSpacing.col * 3, y = y, min = 0, max = 1, vals = { 1, 2 }, table = { [0] = "OFF", "ON" } }
+
+return {
+    read        = 84, -- MSP_OSD_CONFIG
+    write       = 85, -- MSP_SET_OSD_CONFIG
+    title       = "OSD Elements",
+    reboot      = false,
+    eepromWrite = true,
+    minBytes    = 3,
+    labels      = labels,
+    fields      = fields,
+    postLoad = function(self)
+        self.fields[1].value = address
+        self.fields[2].value, self.fields[3].value, self.fields[4].value, self.fields[5].value = self.splitVal(self, self.values[firstVal], self.values[secondVal])
+    end,
+    preSave = function(self)
+        self.values = {}
+        self.values[1] = self.fields[1].value
+        local combineValue = self.checkProfile(self, self.fields[2].value, self.fields[3].value, self.fields[4].value, self.fields[5].value)
+        self.values[2] = bit32.band(combineValue, 0xFF)
+        self.values[3] = bit32.rshift(combineValue, 8)
+        return self.values
+    end,
+    checkProfile = function(self, value, profileFirst, profileSecond, profileThird)
+        local profiles = profileFirst + bit32.lshift(profileSecond, 1) + bit32.lshift(profileThird, 2)
+        local output = bit32.replace(value, profiles, 11, 3)
+        return output
+    end,
+    splitVal = function(self, inputFirstVal, inputSecondVal)
+        local inputVal = inputFirstVal + bit32.lshift(inputSecondVal, 8)
+        local profiles = bit32.extract(inputVal, 11, 3)
+        local fieldPos = bit32.extract(inputVal, 0, 11)
+        local fieldFirstProfile = bit32.band(bit32.rshift(profiles, 0), 1)
+        local fieldThirdProfile = bit32.band(bit32.rshift(profiles, 1), 1)
+        local fieldFourthProfile = bit32.band(bit32.rshift(profiles, 2), 1)
+        return fieldPos, fieldFirstProfile, fieldThirdProfile, fieldFourthProfile
```

Comment:
Logic stays the same but guess:

`FieldFourthProfile` should be `FieldThirdProfile`
`FieldThirdProfile` should be `FieldSecondProfile`

Reply:
lol good catch. 🥇 
@druckgott Please fix 😁

Reply:
I will fix tomorrow

Reply:
It´s fixed